app/src/main/java/com/example/myapplication/CalculatorFragment.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.myapplication
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.example.myapplication.databinding.FragmentCalculatorBinding
+
+
+class CalculatorFragment : Fragment() {
+
+    private var _binding: FragmentCalculatorBinding? = null
+    private val binding: FragmentCalculatorBinding
+        get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentCalculatorBinding.inflate(inflater, container, false)
+            .also { _binding = it }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        with(binding) {
+            button0.setOnClickListener {
+                with(binding) { textView.append((button0.text))}
+            }
```

Comment:
здесь можно не использовать дополнительный `with(binding)`, ты и так в контексте его находишься
```suggestion
            button0.setOnClickListener {
                textView.append(button0.text)
            }
```